packages/contentstack-config/src/utils/rate-limit-handler.ts

```diff
@@ -0,0 +1,55 @@
+import { cliux, configHandler } from '@contentstack/cli-utilities';
+
+const requiredLimitsArray = ['getLimit', 'limit', 'bulkLimit'];
```

Comment:
@harshithad0703 ,
Keep this in an alternative file for usage in the flag option and default flag scenarios.